app-service-template/res/configuration.toml

```
@@ -105,11 +112,20 @@ TokenFile = "/tmp/edgex/secrets/new-app-service/secrets-token.json"
   Port = 59881
 
   [Clients.core-command]
-  Protocol = "http"
-  Host = "localhost"
-  Port = 59882
-
-  [Clients.support-notifications]
+#  Protocol = "http"
+#  Host = "localhost"
+#  Port = 59882
+  # Choose to use new messaging based communication for Command Client.
+  # Only valid if MessageBus Trigger is used
+  # If prefer REST, remove below and uncomment above
+  UseMessageBus = true
+  [Clients.core-command.Topics]
+  QueryRequestTopic = "edgex/core/commandquery/request"
+  QueryResponseTopic = "edgex/core/commandquery/response/#"
```

```suggestion
  QueryResponseTopic = "edgex/core/commandquery/response"
```

This should fix the timeout error